de.metas.business.rest-api/src/main/java/de/metas/rest_api/invoicecandidates/request/JsonInvoiceCandCreateRequest.java

```diff
@@ -0,0 +1,91 @@
+package de.metas.rest_api.invoicecandidates.request;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Singular;
+import lombok.Value;
+
+/*
+ * #%L
+ * de.metas.business.rest-api
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+@Value
+public final class JsonInvoiceCandCreateRequest
+{
+	@ApiModelProperty( //
+			allowEmptyValue = false, //
+			value = "These are the invoices json objects</code>.\n")
+	List<JsonInvoiceCandidate> jsonInvoices;
+
+	@ApiModelProperty(value = "Date of the invoice.")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+	@JsonInclude(Include.NON_NULL)
+	private LocalDate dateInvoiced;
+
+	@ApiModelProperty(value = "Accounting date.")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+	@JsonInclude(Include.NON_NULL)
+	private LocalDate dateAcct;
+
+	@ApiModelProperty(value = "External reference (document number) on a remote system. Not neccesarily unique, but but the external user will want to filter recrods using it")
```

Comment:
Yes, if this is set, it will update C_Invoice_Candidate.POReference of the ICs in question - but the exact meaning of po reference - don't know it, I just took order candidates rest implementation as an example. To be honest I don't know what is the meaning of all these invoicing params.

Reply:
What is **po** coming from? What should I write in the value?

Reply:
POReference can hold is the external customer's purchase order's documentno.
(their purchase order's pendant is our sales order).
So the claim that  but but the external user will want to filter records using it might be right (she can currently do that, in the webui), 
but i think the more relevant info is the one from your screenshot, so i suggest to paste that to the API-doc